pkg/oc/admin/router/router.go

```diff
@@ -915,3 +934,27 @@ func validateServiceAccount(client securityclientinternal.Interface, ns string,
 	}
 	return nil
 }
+
+func getDefaultThreads() int32 {
+	cpuinfo := readProcFile("/proc/cpuinfo")
+	siblings, _ := strconv.Atoi(cpuinfo["siblings"])
+	cores, _ := strconv.Atoi(cpuinfo["cpucores"])
+	threads := siblings * cores
```

Comment:
Don't we want to consider CPU requests/limits specified by the router pod/container?
https://kubernetes.io/docs/tasks/configure-pod-container/assign-cpu-resource/#specify-a-cpu-request-and-a-cpu-limit

Reply:
If the router is running as a privileged container, /proc/cpuinfo will return container or host limits?

Reply:
> If the router is running as a privileged container, /proc/cpuinfo will return container or host limits?

Worse than that, it will be `/proc/cpuinfo` on the host where the administrator runs `oc adm router`, which is probably not the node host.  Calculating the default number of threads really needs to be done in `pkg/router/template/router.go`, not `pkg/oc/admin/router/router.go`.